test/types/AccountManager.test.ts

```diff
@@ -0,0 +1,312 @@
+import type { EnvelopingTypes } from '@rsksmart/rif-relay-contracts/dist/typechain-types/contracts/RelayHub';
+import { expect, use } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import crypto from 'crypto';
+import { providers, Signer, Wallet } from 'ethers';
+import { getAddress } from 'ethers/lib/utils';
+import { createSandbox } from 'sinon';
+import sinonChai from 'sinon-chai';
+import AccountManager from '../../src/AccountManager';
+import * as typedDataUtils from '../../src/typedRequestData.utils';
+
+use(sinonChai);
+use(chaiAsPromised);
+
+const sandbox = createSandbox();
+
+function createRandomAddress() {
+  const randomHex = crypto.randomBytes(20).toString('hex');
+
+  return getAddress(`0x${randomHex}`);
+}
```

Comment:
actually, this might be better using wallet.createWallet when I think about it...